packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -154,6 +155,7 @@ export const HelpCenterContactForm = () => {
 	const overflow = params.get( 'overflow' ) === 'true';
 	const navigate = useNavigate();
 	const [ hideSiteInfo, setHideSiteInfo ] = useState( false );
+	const [ gptFetching, setGPTFetching ] = useState( 'loading' );
```

Comment:
You don't need your own state, you can just duplicate the query and RQ will take care of syncing everything for you.

`const { isFetching } = useJetpackSearchAIQuery( '9619154', message, 'response' )`

Reply:
NICE!

Reply:
@alshakero when trying to implement this, I'm getting the following error:
```
Uncaught Error: No QueryClient set, use QueryClientProvider to set one
    useQueryClient app:///../node_modules/react-query/es/react/QueryClientProvider.js:32
    useBaseQuery app:///../node_modules/react-query/es/react/useBaseQuery.js:21
    useQuery app:///../node_modules/react-query/es/react/useQuery.js:14
    useJetpackSearchAIQuery app:///../packages/data-stores/src/queries/use-jetpack-search-ai.ts:10
    HelpCenterContactForm app:///../packages/help-center/src/components/help-center-contact-form.tsx:428
``` 

Do I need to implement a QueryClient on `useJetpackSearchAIQuery`?

Reply:
This error means there was no `QueryClientProvider` up the tree of React components. And I'm sure it's there because [this](https://github.com/Automattic/wp-calypso/blob/gpt-help-center/packages/help-center/src/components/help-center-contact-form.tsx#L166) works. 

It's also strange that it works inside `HelpCenterGPT` component but not here. 

Did you try this in the editor or in Calypso?

Reply:
That was happening because the import in `use-jetpack-search-ai.ts` was
`import { useQuery } from 'react-query';`
 instead of 
`import { useQuery } from '@tanstack/react-query';`

Fixed in 7d8aa74efe8148d027f4614ac5ede6402aea8c35